source/content/guides/decoupled/overview/02-considerations.md

```diff
@@ -65,6 +65,7 @@ You can use frontend frameworks other than Gatsby and Next.js, however the proce
 - A repo cannot be connected to more than one Front-End Site
 - There are known issues around disconnecting and reconnecting a repo
 - There are known issues around the GitHub app and org level permissions
+- The Front-End Sites cache purging approach is not currently compatible with the Next.js 13 App Router.
```

Comment:
I'm not sure someone landing on this page would have enough context to know what 'The Front-End Sites cache purging approach' is. But I struggle with the right way to elaborate on this. How about:

"The caching recommendations outlined for use with our Next.js starter kits are currently only compatible with the Next.js Pages Router, not the App Router."